clients/src/test/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventProcessorTest.java

```diff
@@ -72,6 +72,7 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
+@SuppressWarnings("ClassDataAbstractionCoupling")
```

Comment:
could you double check if we really need this after the latest changes? (I don't see many new deps in the changes anymore)

Reply:
Yes, we still need this line, or it shows following error.

```
[ant:checkstyle] [ERROR] apache/kafka/clients/src/test/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventProcessorTest.java:75:1: Class Data Abstraction Coupling is 27 (max allowed is 25) classes [ApplicationEventProcessor, AssignmentChangeEvent, AsyncCommitEvent, CheckAndUpdatePositionsEvent, CommitOnCloseEvent, CompletableFuture, CreateFetchRequestsEvent, FetchCommittedOffsetsEvent, ListOffsetsEvent, LogContext, MockRebalanceListener, MockTime, NetworkClientDelegate.PollResult, OffsetAndMetadata, PollEvent, RequestManagers, ResetOffsetEvent, SeekUnvalidatedEvent, SubscriptionState, SubscriptionState.FetchPosition, SyncCommitEvent, TopicMetadataEvent, TopicPartition, TopicPatternSubscriptionChangeEvent, TopicSubscriptionChangeEvent, UnsubscribeEvent, UpdatePatternSubscriptionEvent]. [ClassDataAbstractionCoupling]
```